### ISSUE TITLE
DEPRECATION WARNING: Initialization autoloaded the constants ActionText::ContentHelper and ActionText::TagHelper

### ISSUE THREAD

**Issue by ghiculescu** [bootstrap-email/bootstrap-email#74]
In apps using `bootstrap-email`, this deprecation warning comes up when booting a test console in Rails 6.1.

To replicate:

```
$ rails -v
Rails 6.1.0
$ rails new test-app
$ cd test-app
$ # add bootstrap-email to gemfile
$ bundle install
$ spring stop; bin/rails test
Spring stopped.
DEPRECATION WARNING: Initialization autoloaded the constants ActionText::ContentHelper and ActionText::TagHelper.

Being able to do this is deprecated. Autoloading during initialization is going
to be an error condition in future versions of Rails.

Reloading does not reboot the application, and therefore code executed during
initialization does not run again. So, if you reload ActionText::ContentHelper, for example,
the expected changes won't be reflected in that stale Module object.

These autoloaded constants have been unloaded.

In order to autoload safely at boot time, please wrap your code in a reloader
callback this way:

    Rails.application.reloader.to_prepare do
      # Autoload classes and modules needed at boot time here.
    end

That block runs when the application boots, and every time there is a reload.
For historical reasons, it may run twice, so it has to be idempotent.

Check the "Autoloading and Reloading Constants" guide to learn more about how
Rails autoloads and reloads.
 (called from <main> at /Users/alex/code/test-app/config/environment.rb:5)
```

I think this file is the culprit https://github.com/stuyam/bootstrap-email/blob/master/lib/bootstrap-email/action_mailer.rb

See https://github.com/rails/rails/issues/36546 and linked issues for many other examples.

I think the fix will be similar to [this](https://github.com/mileszs/wicked_pdf/pull/869/files#diff-872e0a26631d6103ce120b7e32313dbec795e73994f865312451a08023cb6a99R9).

**Comment by stuyam** [comment]
Thanks for this, good catch, and great reproduction steps I really appreciate the details. Will take a look. May be a part of the 1.0.0 release of bootstrap email coming in Q1 of 2021, tbd.

**Comment by stuyam** [comment]
Just pushed a new version with a fix for this in v0.3.3
